src/components/TimeCardPage/utils.tsx

```diff
@@ -31,13 +31,15 @@ export const createNewComment = (
 export const createNewReport = (
   user: UserSchema,
   content: ReportOptions,
-  correctTime: number
+  notified: string,
+  explanation: string,
 ) => {
   return {
```

Comment:
I might have missed something up above, but is there a reason why we got rid of the timestamps for the createNewReport function? It looks like the timestamp is still in the `ReportSchema`.